### ISSUE TITLE
Posting Scheduled Transaction Early

### ISSUE THREAD

**Issue by mkemen** [actualbudget/releases#75]
I often post my scheduled transactions a day or two early.   When I select a transaction and choose to post it, it still is appearing in the upcoming scheduled transactions list.   If something is scheduled for 11/18 and I post it on 11/16, it should then set next scheduled date for 12/18.

Perhaps I am doing something wrong.  Help me out.

**Comment by Deguello** [comment]
Agreed. I raised this earlier and the response was something along the lines of (and any misquoting or misremembering is my fault) "the status changes to pending so you know the transaction is in the pipeline and the date will roll over to the next iteration tomorrow." 

I'm not saying that's the wrong functionality, but it's not what I expected. I can't speak for anyone else, but I expected entering the Scheduled transaction to pop that one off the top of the queue and make the next dated instance the one for the subsequent period.

**Comment by ColGra** [comment]
As I understand it, it posts the entry with the scheduled date rather than the current date. Hence the transaction is still classed as pending or upcoming. I guess there are two reasons why you post a schedule in advance - 1) so you can forward plan or 2) because the amount has landed on your account early. If it is item 2) changing the date will remove the upcoming tag and the schedule will roll over. I quite like the current functionality
